apps/events/internal/grpc_impl/operations.go

```diff
@@ -102,7 +104,7 @@ func (c *EventsGrpcImplementation) processFilters(
 }
 
 func (c *EventsGrpcImplementation) processOperations(channelId string, event model.Event, data internal.Data) {
-	streamerApiClient, err := twitch.NewUserClient(channelId, *c.services.Cfg, c.services.TokensGrpc)
+	streamerApiClient, err := twitch.NewUserClient(context.Background(), channelId, *c.services.Cfg, c.services.TokensGrpc)
```

Comment:
There is `NewUserCLientWithContext`

Reply:
not found context to propagate, changed back to `NewUserClient` without context

Reply:
Ok, but revert `twitch.NewUserClient`, because it already has `NewUserClientWithContext`

Reply:
i have already reverted it, check actual diff
![image](https://github.com/Satont/twir/assets/11417502/50146ce6-33c6-48e9-b0d3-ec4174f24b51)